pymedphys/_mosaiq/helpers.py

```diff
@@ -432,12 +440,17 @@ def get_all_treatment_data(cursor, mrn):
             "couch_lat",
             "couch_lng",
             "couch_ang",
+            "tolerance",
+            "time",
         ],
     )
 
     mosaiq_fields.drop_duplicates(inplace=True)
     mosaiq_fields["field_type"] = [
         FIELD_TYPES[item] for item in mosaiq_fields["field_type"]
     ]
+    mosaiq_fields["tolerance"] = [
+        TOLERANCE_TYPES[item] for item in mosaiq_fields["tolerance"]
```

Comment:
You're going to need to remove this from here and just pass the raw numbers out of the function. The conversion can then be down on your side of the function.